### PR TITLE
feat: update squad list

### PR DESCRIPTION
**Pull request by 0xShishigami**
- Add members.
- Update role and description.
- Sort by joining date.

New members: 
- [X] Lempire
- [X] Laruku
- [X] Funkornaut
- [X] Shishigami
- [X] Nigiri
- [X] Yaco
- [x] Ilpepepig
- [X] Zorzal
- [X] Lumi
- [X] Onizuka
- [X] Ringo
- [X] Jahabeebs
- [X] Ikiryou
- [X] Flux

### CODE REVIEW

**Review by turtlemoji — APPROVED** [review]

**Review by 0xtiti — APPROVED** [review]

File: src/data/squad.json [hunk 1]
```diff
@@ -190,5 +190,83 @@
...
+  {
+    "name": "Yaco",
+    "position": "Developer",
+    "image": "/img/pfp/Yaco.png",
+    "description": "- Backend/Data engineer.\n- Enjoy doing stuff that just works.\n\nFun fact: Dulce de batata enjoyer & sudoku puzzle enthusiast."
+  },
+  {
+    "name": "Ilpepepig",
+    "position": "Developer",
+    "image": "/img/pfp/Ilpepepig.png",
```
Comment (0xArdy) [inline src/data/squad.json:233]:
```suggestion
    "image": "/img/pfp/ilpepepig.png",
```

File: src/data/squad.json [hunk 2]
```diff
@@ -190,5 +190,83 @@
...
+    "name": "Funkornaut",
+    "position": "Developer",
+    "image": "/img/pfp/Funkornaut.png",
+    "description": "- Solidity developer.\n- Previously made beer for a living.\n- Big golf guy and yogi.\n\nFun fact: I can wiggle my ears without touching them."
+  },
+  {
+    "name": "Shishigami",
+    "position": "Developer",
+    "image": "/img/pfp/Shishigami.png",
+    "description": "- Designer turned Developer.\n- Obsessed with rendering crazy stuff on displays.\n\n- Fun fact: makes big decisions by talking to plants."
```
Comment (0xArdy) [inline src/data/squad.json:216]:
```suggestion
    "description": "- Designer turned Developer.\n- Obsessed with rendering crazy stuff on displays.\n\nFun fact: makes big decisions by talking to plants."
```

**Review by 0xArdy — CHANGES_REQUESTED** [review]

File: src/data/squad.json (hunk 1 above)
Comment (0xLaruku) [reply src/data/squad.json:233]: I saw this pfp was not loading and its alt text it's "twitter icon", it should be more accurate.

File: src/data/squad.json (hunk 2 above)
Comment (0xShishigami) [reply src/data/squad.json]: https://github.com/defi-wonderland/web/pull/111/commits/59443ccda7b3e5b30fa4307e0b6ce982e99f01da

File: src/data/squad.json (hunk 1 above)
Comment (0xShishigami) [reply src/data/squad.json]: https://github.com/defi-wonderland/web/pull/111/commits/cbf49ea9f3d23fd5a7ceb85f993c859078cb4149

**Review by 0xArdy — APPROVED** [review]
awesome! 🔥

**Review by Onizuka-wl — APPROVED** [review]

**Pull request merged**